ui/static/ui/js/learning_resources.jsx

```diff
@@ -379,12 +386,17 @@ define('learning_resources',
         thiz.setState({
           message: {error: "Unable to save form"}
         });
+      }).always(function() {
+        thiz.setState({loaded: true});
```

Comment:
Curious about the semantics of this. If I understand correctly, setting loaded state to true just removes the spinner, correct? What if we want to tie other events to success in the future? Wouldn't having loaded state set to true (removing the spinner), even when the promise results in a rejection result in our triggering those hypothethical events when we shouldn't? @noisecapella